README.md

```diff
@@ -4,13 +4,39 @@ FFC payment services management
 
 ## Prerequisites
 
+- Azure App Registration Setup
 - Docker
 - Docker Compose
 
 Optional:
 - Kubernetes
 - Helm
 
+## Azure App Registration
+
+`ffc-pay-web` has been integrated into Azure App Registration using the msal-node npm packackage (https://www.npmjs.com/package/@azure/msal-node)
```

Comment:
Typo: `packackage`

I'd also suggest making the link a markdown link.  eg: `[npm package](https://www.npmjs.com/package/@azure/msal-node)`